docs/develop/ruby/set-up-local-ruby.mdx

```diff
@@ -1,49 +1,52 @@
 ---
 id: set-up-local-ruby
-title: Ruby SDK developer guide
-sidebar_label: Quickstart - Set up 
-description: Use the Temporal Ruby SDK to develop Temporal Applications, connect to the Temporal Service, test Workflows and Activities, handle failures, send messages, and more.
-toc_max_heading_level: 4
+title: Set up your local with the Ruby SDK
+sidebar_label: Quickstart - Setup
+description: Configure your local development environment to get started developing with Temporal
 keywords:
-  - ruby
-  - sdk
+  - set up
+  - getting started
+  - ruby sdk
 tags:
   - Ruby SDK
   - Temporal SDKs
+  - setup
+  - getting started
+hide_table_of_contents: true
 ---
 
+import { SetupSteps, SetupStep, CodeSnippet } from "@site/src/components/elements/SetupSteps";
+import { CallToAction } from "@site/src/components/elements/CallToAction";
+
+# Quickstart - Setup
+
 This guide walks you through setting up the Temporal Ruby SDK and running your first Workflow.
 In just a few steps, you'll install the SDK and start a local development server.
 To validate that your local environment is correctly installed, we will execute a Workflow that will output "Hello, Temporal".
 
+<SetupSteps>
```

Comment:
Should there be a step in here that references this? https://learn.temporal.io/getting_started/ruby/dev_environment/

I think the other quick starts have the environment setup.